specs/credentialmanagement/index.src.html

```diff
@@ -495,16 +512,17 @@ <h4 id="interfaces-credential-types-credential"><code>Credential</code></h4>
     interface Credential {
       readonly attribute DOMString id;
       readonly attribute DOMString type;
+      boolean isMatch (any filter);
```

Comment:
I'm misunderstanding the goal of custom credentials, then. I thought you wanted to subclass `Credential` in order to support things like `AgeVerificationCredential` (which I assumed would need to talk to some third-party in order to get data, via the `callback` POST that @dlongley talked about earlier). Is that not the case?

Can you help me understand how you'd intend to use this mechanism?